src/sass/main.scss

```diff
@@ -88,3 +88,30 @@ form {
 .au-form-group + .au-form-group {
 	@include AU-space( 'margin-top', 2unit)
 }
+
+hr {
+	border-color: lightgray !important;
+}
+
+.media-release__caption {
+	color: $AU-color-foreground-muted;
+	@include AU-fontgrid('md');
+}
+
+.media-release__metadata {
```

Comment:
Metadata blocks would be common to multiple content types. Is a `media-release__` prefix required?

Reply:
this is called [BEM](http://getbem.com/naming/)